src/app/map-location/map-location.service.ts

```diff
@@ -15,6 +15,8 @@ export class MapLocationService {
       .set('pageNumber', pageNumber.toString())
       .set('pageSize', pageSize.toString());
 
+    console.log('Route: ' + route.id);
```

Comment:
ja bym chyba usuwał logi przed PR bo potem się ich nazbiera i trudno będzie je zrozumieć. Jak uważasz, ze mogą być to zostaw, a jak nie to usuń wszędzie